carma_wm/src/SignalizedIntersectionManager.cpp

```diff
@@ -143,14 +154,14 @@ namespace carma_wm
     {
       for (auto entry_lane : iter->second)
       {
-        ROS_DEBUG_STREAM("Adding  entry_lane id: " << entry_lane);
+        ROS_DEBUG_STREAM("Adding entry_lane id: " << entry_lane);
         if (entry.find(entry_lane) != entry.end())
         {
           signal_group_to_entry_lanelet_ids_[iter->first].insert(entry[entry_lane]);
         }
         else
         {
-          throw std::invalid_argument(std::string("Unable to convert entry lane Id: " + std::to_string((int)entry_lane) + ", to lanelet id using the given MAP.msg!").c_str());
+          ROS_WARN_STREAM("Unable to convert entry lane Id: "  + std::to_string((int)entry_lane) + ", to lanelet id using the given MAP.msg!");
```

Comment:
Seems that many exceptions have been replaced with warning logs. 
What is the impact of this? If these events occur is the vehicle still safely able to operate?

Reply:
Having `throw exception` there made it really hard to integration test as it would automatically shut every thing down. In worst case scenario, the car will drive through red as it did not detect it as entry into an intersection. But again, for continuing testing it might be better as this.

Reply:
Can you put up an enhancement issue to review this and add a TODO with the link here as something we should come back to review.

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/issues/1593
Enhancement issue added.